skrf/media/rectangularWaveguide.py

```diff
@@ -121,9 +121,10 @@ def __repr__(self):
 
     
     @classmethod
-    def from_Z0(cls,frequency, Z0,f, m=1,n=0,ep_r=1, mu_r=1, **kw):
+    def from_Z0(cls,frequency, Z0,f, ep_r=1, mu_r=1, **kw):
```

Comment:
I know it was `from_Z0` in the existing code, but it's time to consider renaming it to `from_z0` to keep consistency with the convention adopted in scikit-rf.

Reply:
I will do it with the next commit